applications/luci-app-filemanager/htdocs/luci-static/resources/view/system/filemanager.js

```diff
@@ -0,0 +1,1571 @@
+                '.file-manager-header input { margin-left: 10px; width: 100%; max-width: 700px; font-size: 18px; }' +
+                '.file-manager-header button { margin-left: 10px; font-size: 18px; }'
+            ),
+            E('div', { 'class': 'cbi-tabcontainer', 'id': 'tab-group' }, [
+                E('ul', { 'class': 'cbi-tabmenu' }, [
+                    E('li', { 'class': 'cbi-tab cbi-tab-active', 'id': 'tab-filemanager' }, [
+                        E('a', { 'href': '#', 'click': this.switchToTab.bind(this, 'filemanager') }, _('File Manager'))
+                    ]),
+                    E('li', { 'class': 'cbi-tab', 'id': 'tab-editor' }, [
+                        E('a', { 'href': '#', 'click': this.switchToTab.bind(this, 'editor') }, _('Editor'))
+                    ]),
+                    E('li', { 'class': 'cbi-tab', 'id': 'tab-settings' }, [
+                        E('a', { 'href': '#', 'click': this.switchToTab.bind(this, 'settings') }, _('Settings'))
+                    ])
+                ])
+            ]),
+            E('div', { 'class': 'cbi-tabcontainer-content' }, [
+                E('div', { 'id': 'content-filemanager', 'class': 'cbi-tab', 'style': 'display:block;' }, [
+                    E('div', { 'id': 'file-list-container','class': 'resizable', 'style': 'width: ' + config.windowSizes.width + 'px; height: ' + config.windowSizes.height + 'px;' }, [
+                        E('table', { 'class': 'table', 'id': 'file-table' }, [
+                            E('thead', {}, [
+                                E('tr', {}, [
+                                    E('th', { 'data-field': 'name' }, [
+                                        _('Name'),
+                                        E('button', { 'class': 'sort-button', 'data-field': 'name', 'title': 'Sort by Name' }, '↕'),
+                                        E('div', { 'class': 'resizer' })
+                                    ]),
+                                    E('th', { 'data-field': 'type' }, [
+                                        _('Type'),
+                                        E('button', { 'class': 'sort-button', 'data-field': 'type', 'title': 'Sort by Type' }, '↕'),
+                                        E('div', { 'class': 'resizer' })
+                                    ]),
+                                    E('th', { 'data-field': 'size' }, [
+                                        _('Size'),
+                                        E('button', { 'class': 'sort-button', 'data-field': 'size', 'title': 'Sort by Size' }, '↕'),
+                                        E('div', { 'class': 'resizer' })
+                                    ]),
+                                    E('th', { 'data-field': 'mtime' }, [
+                                        _('Last Modified'),
+                                        E('button', { 'class': 'sort-button', 'data-field': 'mtime', 'title': 'Sort by Last Modified' }, '↕'),
```

Comment:
All of these title strings should be _('...') i18n able.

Reply:
Fixed